readme.md

```diff
@@ -102,10 +197,31 @@ Temporal is a [durable execution system](https://temporal.io/blog/building-relia
 ### Libraries
 
 - [`lorensr/temporal-time-utils`](https://github.com/lorensr/temporal-time-utils) - `sleepUntil` and `UpdatableTimer`.
+- [Cognosis AI Platform](https://github.com/cognosisai/platform) - Template for LLM applications.
+
+### Tutorials
+
+- [Getting started](https://learn.temporal.io/getting_started/typescript/dev_environment/)
+- [Next.js](https://learn.temporal.io/tutorials/typescript/nextjs/)
+- [Subscription](https://learn.temporal.io/tutorials/typescript/subscriptions/)
```

Comment:
might super briefly explain what subscription is in the context of temporal? (but if it's a common typescript term please disregard as I really only barely know typescript)